Gruntfile.js

```diff
@@ -64,15 +64,11 @@ module.exports = function(grunt) {
       src: ["license_header.txt", "plottable.js"],
       dest: "plottable.js"
     },
-    svgtypewriter: {
```

Comment:
We need to keep concatting this. Imo until we Plottable 3.0.0 and convert everything to modules, we need to build an svgtypewriter bundle file and concat it (or use webpack and create a plottable.js bundle, which is equivalent)